db/msqur-4.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE `metadata` 
```

Comment:
I am pretty sure something like this is probably already done to Nick's local database as inserting a new tune threw a fit about some of the columns not being nullable

Reply:
It's very possible some columns aren't up to date. However, this makes all fields nullable, and that's not what I want. I'd prefer if you removed this file. I can do a schema dump and reset the DB scripts to that.